src/libraries/icubmod/embObjMotionControl/eomcParser.cpp

```diff
@@ -2457,6 +2457,115 @@ bool Parser::parseImpedanceGroup(yarp::os::Searchable &config,std::vector<impeda
 
 }
 
+/*
+typedef struct
+{
+    double Km;
+    double Kw;
+    double S0; 
+    double S1;
+    double Vth;
+    double Fc_pos;
+    double Fc_neg;
+    double Fs_pos;
+    double Fs_neg;
+    double tbd0;
+    double tbd1;
+    double tbd2;
+} lugreParameters_t;
+*/
+
+bool Parser::parseLugreGroup(yarp::os::Searchable &config,std::vector<lugreParameters_t> &lugre)
+{
+    Bottle lugreGroup;
+    lugreGroup=config.findGroup("LUGRE","LUGRE parameters");
+
+    if(lugreGroup.isNull())
+    {
+        for (int j = 0; j<_njoints; ++j)
+        {
+            lugre[j].S0 = 0.0;
+        }
+        
```

Comment:
@ale-git: In case of missing `LUGRE` parameters, which is the correct initialization? When you check the return value of this function, you initialize all `km` fields to -1.0.

Reply:
You're right, it must be Km = -1

Reply:
You're right, it must be Km = -1

Reply:
@ale-git , you set 1.0 and not -1.0. Are you sure?

Reply:
Yes it must be -1. Maybe I need some holidays...

Reply:
solved in the commit f1dc2ce4f10995e60f54f1e3a6cf2ebe36109704. Thanks @ale-git